apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -38,10 +38,12 @@ export default class DetailViewContents extends React.Component {
       email: PropTypes.string,
       form_data: PropTypes.object,
       application_type: PropTypes.oneOf(['Facilitator', 'Teacher']),
-      response_scores: PropTypes.object
+      response_scores: PropTypes.object,
+      meets_criteria: PropTypes.string,
+      bonus_points: PropTypes.number
     }),
-    updateProps: PropTypes.func.isRequired,
-    viewType: PropTypes.oneOf(['teacher', 'facilitator']).isRequired
+    viewType: PropTypes.oneOf(['teacher', 'facilitator']).isRequired,
+    loadFunction: PropTypes.func.isRequired
```

Comment:
nit: This reads like it's going to load a function. How about something like `onRequestData` or `onReload`?

Reply:
I'm not sure about either of those names - this function is supposed to reload the data (without displaying the spinner) but I see your point. Let me think